lnrpc/rpc.proto

```diff
@@ -992,7 +992,7 @@ message OpenChannelRequest {
     uint32 remote_csv_delay = 10 [json_name = "remote_csv_delay"];
 
     /// The maximum amount of coins in millisatoshi that can be pending within the channel.
-	int64 MaxValueInFlight = 11 [json_name = "max_value_in_flight_msat"];
+    int64 MaxValueInFlight = 11 [json_name = "max_value_in_flight_msat"];
```

Comment:
This should be a fixup to the previous `lnrpc` commit :)

Reply:
it is in the right commit now :)